slow_statement_logger/__init__.py

```diff
@@ -7,13 +7,14 @@
 
 import psycopg2
 
-from odoo import sql_db
-from odoo.tools import config
+from openerp import sql_db
+from openerp.tools import config
 
 _logger = logging.getLogger(__name__)
 
 # TODO Odoo option? server_environment?
-LOG_MIN_DURATION_STATEMENT = int(config.get("log_min_duration_statement", "-1"))
+LOG_MIN_DURATION_STATEMENT = int(
+    config.get("log_min_duration_statement", "-1"))
```

Comment:
@ThomasBinsfeld out of curiosity, is there a reason to wrap this line? Normally MQT accepts 88 characters on all branches now.

Reply:
Oops, i thought it was only on 13.0 branch. Fixed